File: software/examples/csharp/ExampleSimple.cs
using System;
using Tinkerforge;

class Example
{
	private static string HOST = "localhost";
	private static int PORT = 4223;
	private static string UID = "XYZ"; // Change XYZ to the UID of your UV Light Bricklet 2.0

	static void Main()
	{
		IPConnection ipcon = new IPConnection(); // Create IP connection
		BrickletUVLightV2 uvl = new BrickletUVLightV2(UID, ipcon); // Create device object

		ipcon.Connect(HOST, PORT); // Connect to brickd
		// Don't use device before ipcon is connected

		// Get current UV-A
		int uva = uvl.GetUVA();
		Console.WriteLine("UV-A: " + uva/10.0 + " mW/m²");

		// Get current UV-B
		int uvb = uvl.GetUVB();
		Console.WriteLine("UV-B: " + uvb/10.0 + " mW/m²");

		// Get current UV index
		int uvi = uvl.GetUVI();
		Console.WriteLine("UV Index: " + uvi/10.0);

		Console.WriteLine("Press enter to exit");
		Console.ReadLine();
		ipcon.Disconnect();
	}
}

File: software/examples/java/ExampleSimple.java
import com.tinkerforge.IPConnection;
import com.tinkerforge.BrickletUVLightV2;

public class ExampleSimple {
	private static final String HOST = "localhost";
	private static final int PORT = 4223;

	// Change XYZ to the UID of your UV Light Bricklet 2.0
	private static final String UID = "XYZ";

	// Note: To make the example code cleaner we do not handle exceptions. Exceptions
	//       you might normally want to catch are described in the documentation
	public static void main(String args[]) throws Exception {
		IPConnection ipcon = new IPConnection(); // Create IP connection
		BrickletUVLightV2 uvl = new BrickletUVLightV2(UID, ipcon); // Create device object

		ipcon.connect(HOST, PORT); // Connect to brickd
		// Don't use device before ipcon is connected

		// Get current UV-A
		int uva = uvl.getUVA(); // Can throw com.tinkerforge.TimeoutException
		System.out.println("UV-A: " + uva/10.0 + " mW/m²");

		// Get current UV-B
		int uvb = uvl.getUVB(); // Can throw com.tinkerforge.TimeoutException
		System.out.println("UV-B: " + uvb/10.0 + " mW/m²");

		// Get current UV index
		int uvi = uvl.getUVI(); // Can throw com.tinkerforge.TimeoutException
		System.out.println("UV Index: " + uvi/10.0);

		System.out.println("Press key to exit"); System.in.read();
		ipcon.disconnect();
	}
}
